mobile/examples/super_resolution/README.md

```diff
@@ -0,0 +1,11 @@
+# ONNX Runtime Mobile Super Resolution Android sample application with Ort-Extensions support for pre/post processing
```

Comment:
Given we're also going to add the same example app for other platforms it might be better to put the android example under mobile/examples/super_resolution/android